packages/office-ui-fabric-react/tslint.json

```diff
@@ -4,6 +4,7 @@
     "deprecation": false,
     "jsx-ban-props": false,
     "no-any": false,
-    "typedef": [false]
+    "typedef": [false],
+    "import-blacklist": [true, { "../../Styling": ["FontSizes"] }]
```

Comment:
@ecraig12345 @KevinTCoughlin I found this rule to work.
I am scoping it to only OUFR for now as I am only fixing it in here and throughout our components we are supposed to use `../../Styling` import. Let me know if that works.